StPetersburg/InEtn210320.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+     xml:lang="en"
+     xml:id="InEtn210320"
+     type="mss">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Prayer to Jesus Christ, Doctrine of Mysteries</title>
+            <editor key="DE"/>
+            <editor role="generalEditor" key="AB"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed
+            under the Creative Commons Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0524InEtn"/>
+                  <idno>InEtn 2103-20</idno>
+                  <altIdentifier>
+                     <idno>Platonov cat. III.107</idno>
+                  </altIdentifier>
+               </msIdentifier>
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="1v" to="5r"/>
+                     <title type="incomplete" ref="LIT5831PrayerJesus"/>
+                     <textLang mainLang="gez"/>
+                    <incipit xml:lang="gez">ይዌድስ፡ ወቦቱ፡ ኵሉ፡ ማኅበረ፡ ፍትሱማን፡</incipit>
```

Comment:
If it is ፍትሱማን፡ and not ፍጹማን in the catalogue/manuscript, maybe mark up with sic?

Reply:
that is a mistake, I have a new keyman :)